apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenChart/TokenChartToolBar.tsx

```diff
@@ -21,7 +21,7 @@ export const TokenChartToolbar = ({timeInterval, disabled, onChange}: Props) =>
 
   return (
     <View style={styles.chartToolbar}>
-      {Object.values(TOKEN_CHART_INTERVAL).map((itv) => (
+      {Object.values(TokenChartInterval).map((itv) => (
```

Comment:
I think it is missing translations it'd show `24 H`, `1 W` for all languages.

Reply:
the interval label from TokenPerformance maybe should be moved to `useStrings` 🤷🏻

Reply:
Mm yeah, that was never translated.

Reply:
create a task please